plugins/modules/virt_volume.py

```diff
@@ -0,0 +1,711 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Maciej Delmanowski <drybjed@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: virt_volume
+author:
+    - Leonardo Galli (@galli-leo)
+short_description: Manage libvirt volumes inside a storage pool
+description:
+    - Manage I(libvirt) volumes inside a storage pool.
+options:
+    name:
+        aliases: [ "volume" ]
```

Comment:
```suggestion
```
Only `name` is ok here, there's a recommendation in the docs to avoid aliases whenever possible

Reply:
I think, the aliases are a good idea here, because in playbooks with lots of tasks from `community.libvirt` it improves the readability, to not have every task contain a `name` which references to a totally different kind of structure.